specification/swlifecycle.adoc

```diff
@@ -629,6 +656,15 @@ Confidential-Unassigned state.
 and returns the PA as an Non-Conf page to the VMM.
 * VMM translations to the PA (via 1st or G stage mappings) may be created now.
 
+[Note]
+====
+For deployment model 3 when no Smmtt is available memory is dynamically partitions. TVM shutdown is
+simler, the VMM can cause all pages to be reclaimed via the sbi_covh_reclaim_pages which will 
```

Comment:
nit: simler/simpler ?

Reply:
Hi Atish, you are reviewing the outdated version of the spec. These typos have been fixed.